Jellyfin.Server/StartupOptions.cs

```diff
@@ -0,0 +1,41 @@
+using CommandLine;
+using Emby.Server.Implementations;
+
+namespace Jellyfin.Server
+{
+    /// <summary>
+    /// Class used by CommandLine package when parsing the command line arguments.
+    /// </summary>
+    public class StartupOptions : IStartupOptions
+    {
+        [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
+        public string DataDir { get; set; }
+
+        [Option('c', "configdir", Required = false, HelpText = "Path to use for config data (user policies and puctures).")]
+        public string ConfigDir { get; set; }
+
+        [Option('l', "logdir", Required = false, HelpText = "Path to use for writing log files.")]
+        public string LogDir { get; set; }
+
+        [Option("ffmpeg", Required = false, HelpText = "Path to external FFmpeg exe to use in place of built-in.")]
+        public string FFmpegPath { get; set; }
+
+        [Option("ffprobe", Required = false, HelpText = "ffmpeg and ffprobe switches must be supplied together.")]
```

Comment:
Help is not that much helpful... :) Maybe reword it somehow?

Reply:
My comment here was driven by observation in code where these switches are used.  If both --ffmpeg and --ffprobe are not provided, together, then which ever is given fails (silently) to be used.  Maybe we ignore that issue for now and revisit another time?

"Path to external FFprobe executable to use in place of default found in PATH."

Reply:
Maybe like this?
`Path to external FFprobe executable to use in place of default found in PATH. Must be specified along with FFmpeg`

Reply:
Changed to above.